integration/raft/config_test.go

```diff
@@ -510,22 +510,25 @@ var _ = Describe("EndToEnd reconfiguration and onboarding", func() {
 				Eventually(ordererProc.Ready(), network.EventuallyTimeout).Should(BeClosed())
 			}
 
+			By("Creating a new channel, joining all orderers")
+			channelparticipation.JoinOrderersAppChannelCluster(network, "testchannel", network.Orderers...)
```

Comment:
Can we have the channel name in a variable & reference same variable in all other statements.

Reply:
I actually prefer it this way, having an explicit "testchannel" helps in readability. Moreover, this is the style in 90% of the integration tests so it is a bit late in the game to try and impose a different style.